site/index.html

```diff
@@ -10,10 +10,10 @@
       </div>
       <div class="col-md-6 order-md-1 text-center text-md-left pr-md-5">
         <h1 class="mb-3 bd-text-purple-bright">Bootstrap</h1>
-        <p class="lead">
+        <p class="bd-lead">
```

Comment:
`lead` has a a size of 1.25 while bd-lead 1.125, plus a max-width. Not sure if we should change to `bd-lead`.

Reply:
I changed this because the `.lead` didn't rescale along with the title. Since the base font size is increased, this didn't matter anymore, so I reverted this change.